src/QMCWaveFunctions/PlaneWave/PWParameterSet.h

```diff
@@ -98,7 +99,7 @@ struct PWParameterSet : public MPIObjectBase
   std::string getCenterName(const std::string& hg, int ib);
   std::string getOriginName(const std::string& hg, int ib);
 
-  void writeParameters(hid_t gid);
+  void writeParameters(hdf_archive& gid);
```

Comment:
The last commit is missing deleting this line

Reply:
fixed :man_facepalming: